app/helpers/application_helper/button/vm_publish.rb

```diff
@@ -2,6 +2,6 @@ class ApplicationHelper::Button::VmPublish < ApplicationHelper::Button::Basic
   needs :@record
 
   def visible?
-    @record.is_available?(:publish) && !@is_redhat
+    @record.supports?(:publish)
```

Comment:
really nice :+1: 

Now, I think we could even get rid of Button `VmPublish` 
@romanblanco is that true? Or should this be done in another PR?